fe/fe-core/src/test/resources/sql/tpchcost/q7.sql

```diff
@@ -116,7 +116,7 @@ OutPut Exchange Id: 22
 |  equal join conjunct: [39: C_NATIONKEY, INT, false] = [50: N_NATIONKEY, INT, false]
 |  other join predicates: ((46: N_NAME = 'CANADA') AND (51: N_NAME = 'IRAN')) OR ((46: N_NAME = 'IRAN') AND (51: N_NAME = 'CANADA'))
 |  build runtime filters:
-|  - filter_id = 3, build_expr = (50: N_NATIONKEY), remote = true
+|  - filter_id = 3, build_expr = (50: N_NATIONKEY), remote = false
```

Comment:
Why remote change from true to false?

Reply:
it is something wrong happens. I will check